### ISSUE TITLE
disableRequestLogging disables an error log in onResponse hook

### ISSUE THREAD

**Issue by fredmaggiowski** [fastify/fastify#2319]
## Question

By reading the doc about [disableRequestLogging](https://www.fastify.io/docs/latest/Server/#disablerequestlogging) option I've understood that the two default logs `incoming request` and `request completed` are disabled but reading the code there's actually another log that gets disabled at [this line](https://github.com/fastify/fastify/blob/667d4644eab09f27bc30cad6e7378f5fc82cb9e8/lib/reply.js#L587)

Is this an intended behaviour? Maybe should be better specified in the docs or do you think it's superfluous?

Thanks

## Environment

- *node version*: 10, 12, 13
- *fastify version*: >=2.0.0
- *os*: Mac

**Comment by mcollina** [comment]
> Is this an intended behaviour? Maybe should be better specified in the docs or do you think it's superfluous?

Yes, it is intended. Would you mind to send a PR to make the docs more clear?

**Comment by fredmaggiowski** [comment]
Alright, I will do it asap, thanks for the prompt response

**Comment by francisdb** [comment]
@fredmaggiowski have you used a workaround to still somewhere report these `onResponse` errors when `disableRequestLogging` is enabled?

**Comment by fredmaggiowski** [comment]
> @fredmaggiowski have you used a workaround to still somewhere report these `onResponse` errors when `disableRequestLogging` is enabled?

Hi @francisdb we were fine with it being disabled so we didn't take any action
